src/Pact/Types/Term.hs

```diff
@@ -608,41 +754,83 @@ data Term n =
     , _tMeta :: !Meta
     , _tInfo :: !Info
     }
-    deriving (Functor,Foldable,Traversable,Eq)
-
-instance Show n => Show (Term n) where
-    show TModule {..} =
-      "(TModule " ++ show _tModuleDef ++ " " ++ show (unscope _tModuleBody) ++ ")"
-    show (TList bs _ _) = "[" ++ unwords (map show bs) ++ "]"
-    show TDef {..} = show _tDef
-    show TNative {..} =
-      "(TNative " ++ asString' _tNativeName ++ " " ++ showFunTypes _tFunTypes ++ " " ++ unpack _tNativeDocs ++ ")"
-    show TConst {..} =
-      "(TConst " ++ asString' _tModule ++ "." ++ show _tConstArg ++ " " ++ show _tMeta ++ ")"
-    show (TApp a _) = show a
-    show (TVar n _) = "(TVar " ++ show n ++ ")"
-    show (TBinding bs b c _) = "(TBinding " ++ show bs ++ " " ++ show (unscope b) ++ " " ++ show c ++ ")"
-    show (TObject bs _ _) =
-      "{" ++ intercalate ", " (map (\(a,b) -> show a ++ ": " ++ show b) bs) ++ "}"
-    show (TLiteral l _) = show l
-    show (TGuard k _) = show k
-    show (TUse u _) = show u
-    show (TValue v _) = BSL.toString $ encode v
-    show (TStep ent e r _) =
-      "(TStep " ++ show ent ++ " " ++ show e ++ maybeDelim " " r ++ ")"
-    show TSchema {..} =
-      "(TSchema " ++ asString' _tModule ++ "." ++ asString' _tSchemaName ++ " " ++
-      show _tFields ++ " " ++ show _tMeta ++ ")"
-    show TTable {..} =
-      "(TTable " ++ asString' _tModule ++ "." ++ asString' _tTableName ++ ":" ++ show _tTableType
-      ++ " " ++ show _tMeta ++ ")"
+    deriving (Functor,Foldable,Traversable,Eq,Show)
+
+-- | Newtype to avoid introducing an orphan instance for @Pretty Var@. Used
+-- just for pretty printing (@prettyUnscope@).
+newtype PrettyVar a b = PrettyVar (Var a b)
+
+instance (Pretty a, Pretty b) => Pretty (PrettyVar a b) where
+  pretty (PrettyVar (B b)) = pretty b
+  pretty (PrettyVar (F a)) = pretty a
+
+prettyUnscope
+  :: (Functor f, Pretty (f (PrettyVar b (f a))))
+  => Scope b f a -> Doc
+prettyUnscope = pretty . fmap PrettyVar . unscope
+
+instance Pretty n => Pretty (Term n) where
+  pretty = \case
+    TModule{..} -> parensSep
+      [ "TModule", pretty _tModuleDef, prettyUnscope _tModuleBody ]
+    TList{..} -> prettyList _tList
+    TDef{..} -> pretty _tDef
+    TNative{..} -> dullgreen ("native function `" <> pretty (asString' _tNativeName) <> "`")
+      <> nest 2 (
+         line
+      <> line <> pretty _tNativeDocs
+      <> line
+      <> line <> dullgreen "Examples:"
+      <> line <> align (vsep (pretty <$> _tNativeExamples))
```

Comment:
Because these are their own separate list now, we can strip the backticks

Reply:
Indeed, in the [gist](https://gist.github.com/joelburget/06debb85be91e4cf46df673951f7e158) I wrote to mock this up, it looks like

```
> (+ 1 2)
> (+ 5.0 0.5)
> (+ "every" "body")
> (+ [1 2] [3 4])
> (+ { "foo": 100 } { "foo": 1, "bar": 2 })
```

but I punted on this yesterday. I'll implement it today.

Reply:
fixed 

![colored-examples](https://user-images.githubusercontent.com/310981/51931482-c0a91200-23b1-11e9-832b-632db02fac71.png)

Reply:
It occurred to me that this doesn't do a very good job of communicating that the thing in red is an error. I'll fix this.